clusterman/autoscaler/autoscaler.py

```diff
@@ -152,6 +152,17 @@ def run(self, dry_run: bool = False, timestamp: Optional[arrow.Arrow] = None) ->
             exception, tb = e, traceback.format_exc()
 
         logger.info(f'Signal {signal_name} requested {resource_request}')
+
+        no_scale_down = False
+        if self.autoscaling_config.prevent_scale_down_after_capacity_loss:
+            removed_nodes_since_last_reload = self.pool_manager.get_removed_nodes_since_last_reload()
+            # TODO (jmalt, CLUSTERMAN-553): we may want something more sophisticated than len(nodes),
```

Comment:
Can we file a new ticket here that's specific to "making this use weights"?

Reply:
Filed CLUSTERMAN-576